datalab/train_subset.ipynb

```diff
@@ -0,0 +1,501 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Classifier Training on Caltech-256 Subset\n",
+    "\n",
+    "In this notebook, we will train a Swin Transformer model for image classifier on a subset of the Caltech-256 dataset using the Timm library and Pytorch. The main steps include dataset sampling and precrocessing, model training with k-fold cross-validation, and feature extraction.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import shutil\n",
+    "import random\n",
+    "\n",
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torch.optim as optim\n",
+    "import torchvision.transforms as transforms\n",
+    "import numpy as np\n",
+    "from torchvision.datasets import Caltech256, ImageFolder\n",
+    "from tqdm import tqdm\n",
+    "import timm\n",
+    "from sklearn.model_selection import StratifiedKFold\n",
+    "\n",
+    "from PIL import Image"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dataset Preparation\n",
```

Comment:
If it's easy to do, I'd skip this entire data prep section and just load the prepped dataset from:

```
!wget -nc 'https://cleanlab-public.s3.amazonaws.com/Datalab/caltech256-subset.tar.gz'
```

This dataset prep section doesn't really help somebody better understand how to train an image classifier on their own data which is the purpose of this notebook

Reply:
If not easy to change though, then just keep as is.

Reply:
Easy enough.